triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala

```diff
@@ -272,6 +284,20 @@ trait AbstractTriggerServiceTestHelper
     eventually {
       pred(getTriggerStatus(triggerInstance).map(_._2))
     }
+
+  def assertTriggerRunnerStatus(
+      triggerInstance: UUID,
+      pred: Vector[String] => Assertion,
+  ): Assertion = {
+    val filterString = s"id: \"$triggerInstance\""
```

Comment:
The log marker (in the log event) holds our trigger logging context. We use this knowledge to match on the trigger ID which will be present within this logging context (and so log marker).